src/python/convert.py

```diff
@@ -1,9 +1,13 @@
 import database
 import fetch
 import whisper
+import utils
 
 def convert():
     transcribed = 0
+    utils.create_file_path(utils.IMAGE_PATH)
+    utils.create_file_path(utils.AUDIO_PATH)
```

Comment:
I think these might actually make a little more sense split up and put inside each of the download functions. My reasoning is two-fold. 

1. This won't needlessly run if there aren't any new RSS episodes.
2. It will colocate the folder creating code right next to the downloading code that uses it.